app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,75 @@
+
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
+// recommended for single sections: auto
+// recommended for longer questions: 50%
+
 .check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+  .set {
+    position: relative;
+    border-bottom: 1px solid $border-colour;
```

Comment:
We have four(ish) options, the choice of which is probably not super important for the prototype kit but will become more important for Elements:

**Child combinator** (what I had before)
```
.foo {
  > * {
  }
}
```
I personally find that quite resilient, especially because every child **has** to be styled in a certain way because the `display: table` children need to be `display: table-row`, etc.
(I wonder if I should even go as far and use `:first-child` for the question and `:last-child` for the change link.)
But I understand that people don't like this.

**Simple classes** (what I have now)
```
.foo {
  .bar {
  }
}
```
I agree with Nick that the class names are too generic.

**Prefixed classes** (BEM would also fit here, although the example below is not BEM)
```
.govuk-foo {
  .govuk-foo-bar {
  }
}
```
Probably where we're generally going? I personally don't like it but that's another story.

**Simple classes with child combinator**
```
.foo {
  > .bar {
  }
}
```
More resilient than just simple classes on their own. But not resilient enough to stop potentially wrongly global classes to have an effect.